packages/pn-commons/src/hooks/useIsCancelled.ts

```diff
@@ -0,0 +1,23 @@
+import { NotificationDetail, NotificationStatus, TimelineCategory } from '../types';
+
+/**
+ * Checks if notification is cancelled
+ */
+
+interface Props {
+  notification: NotificationDetail;
+}
+
+export const useIsCancelled = ({ notification }: Props) => {
```

Comment:
@leleOFA add some jsdoc to this hook

Reply:
do it

Reply:
do it

Reply:
@leleOFA jsdoc should prefix the hook declaration. move them to line 15

Reply:
do it